ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/page/v3/page/_cq_dialog/.content.xml

```diff
@@ -141,6 +141,38 @@
                                                     sling:resourceType="core/cif/components/renderconditions/pagetype"
                                                     pageType="landing"/>
                                             </myAccountPage>
+                                            <clearCache
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/container">
+                                                <granite:rendercondition
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="core/cif/components/renderconditions/pagetype"
+                                                        pageType="landing"/>
+                                                <granite:rendercondition
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="core/cif/components/renderconditions/dispatcherConfigCondition"
+                                                />
+                                                <items jcr:primaryType="nt:unstructured">
+                                                    <label
+                                                            jcr:primaryType="nt:unstructured"
+                                                            sling:resourceType="granite/ui/components/coral/foundation/text"
+                                                            granite:class="cif-clear-cache-label"
+                                                            text="Clear the full cache of the corresponding site">
+                                                    </label>
+                                                    <button
+                                                            jcr:primaryType="nt:unstructured"
+                                                            sling:resourceType="granite/ui/components/coral/foundation/button"
+                                                            text="Clear Cache"
+                                                            variant="primary"
```

Comment:
Is suggest variant="secondary" to look more like the cancel button in the action bar rather than the primary "save&close" button.